src/module.cc

```diff
@@ -46,7 +46,7 @@ bool Module::matchesRequestPath(const std::string& reqPath) const {
         pathSize--;
     }
 
-    for (int i = 0; i < pathSize; ++i) {
+    for (size_t i = 0; i < pathSize; ++i) {
```

Comment:
The reason you were previously getting an error on the int type was probably that it was not big enough to contain the object. With size_t the good thing is that it is at least is as big as the biggest size required. Maybe you could try using a unsigned long? Would be more efficient perhaps.